prince-of-versions/src/main/java/co/infinum/princeofversions/helpers/parsers/JsonVersionConfigParser.java

```diff
@@ -224,4 +237,35 @@ private Version extractMinimumVersion(JSONObject data) throws JSONException {
         }
         return metadata;
     }
+
+    private int extractLastMinSdk(JSONObject data) throws JSONException {
+        int lastMinSdk;
+        if (data.has(ANDROID)) {
+            JSONObject android = data.getJSONObject(ANDROID);
+            if (android.has(LAST_MIN_SDK)) {
+                lastMinSdk = android.getInt(LAST_MIN_SDK);
+                if (lastMinSdk != 0) {
+                    return lastMinSdk;
+                }
+            }
+        }
+        return VersionContext.UpdateContext.DEFAULT_MIN_SDK_VALUE;
+    }
+
+    private int extractNewMinSdk(JSONObject data) throws JSONException {
+        int newMinSdk;
+        if (data.has(ANDROID)) {
+            JSONObject android = data.getJSONObject(ANDROID);
+            if (android.has(LATEST_VERSION)) {
+                JSONObject latestVersion = android.getJSONObject(LATEST_VERSION);
+                if (latestVersion.has(NEW_MIN_SDK)) {
+                    newMinSdk = latestVersion.getInt(NEW_MIN_SDK);
```

Comment:
This will fail if `NEW_MIN_SDK` field cannot be converted to int. Probably we want to catch that exception and fallback to `VersionContext.UpdateContext.DEFAULT_MIN_SDK_VALUE`. What do you think?

Same comment for `extractLastMinSdk` method.